src/utility/diffable_threads.jl

```diff
@@ -50,14 +50,17 @@ or equivalently with
 ```
 set_scheduler!(:static; ntasks=4, chunking=true)
 ```
+If no `scheduler` is passed and only kwargs are provided, the `DynamicScheduler`
+constructor is used with the provided kwargs.
 For a detailed description of all schedulers and their keyword arguments consult the
 [`OhMyThreads` documentation](https://juliafolds2.github.io/OhMyThreads.jl/stable/refs/api/#Schedulers).
 
 To reset the scheduler to its default value, one calls `set_scheduler!` without passing
 arguments which then uses the default `DynamicScheduler()`. If the number of used threads is
 just one it falls back to `StaticScheduler()`.
 """
-set_scheduler!(scheduler; kwargs...) = Defaults.set_scheduler!(scheduler; kwargs...)
+set_scheduler!(scheduler=OhMyThreads.Implementation.NotGiven(); kwargs...) =
+    Defaults.set_scheduler!(scheduler; kwargs...)
```

Comment:
why not just export the `set_scheduler` function from `Defaults`?

Reply:
Oh sure, not sure why I did it this way initially